src/components/Menu/MenuBar/MenuBar.vue

```diff
@@ -191,7 +192,7 @@ watch(projectFilePath, updateRecentProjects, {
 });
 
 // 「エンジン」メニューのエンジン毎の項目
-const engineSubMenuData = computed<MenuItemData[]>(() => {
+const engineSubMenuData = computed(() => {
```

Comment:
前は型があって、そっちのが良さそうだったので
```suggestion
const engineSubMenuData = computed<MenuItemData[]>(() => {
```